iotdb-core/datanode/src/main/java/org/apache/iotdb/db/storageengine/dataregion/wal/node/WALNode.java

```diff
@@ -252,20 +254,24 @@ public DeleteOutdatedFileTask() {
     }
 
     private void init() {
-      this.firstValidVersionId = initFirstValidWALVersionId();
```

Comment:
We don‘t need firstValidWALVersionId any more, but I think we should keep initFirstValidWALVersionId method because the delete procedure will try to delete current file in this method.